### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by zdrale**
Clone of The NextWeb webpage with a responsive layout. The project contains the following components.

Used flex, grid, and float for alignment and layout
Used semantic HTML for header, main content, and footer

### CODE REVIEW

File: Assets/style.css [hunk 1]
```diff
@@ -0,0 +1,713 @@
...
+}
+
+/* Utility classes */
+.p-0 {
+  padding: 0 !important;
+}
+
+.fw-bold {
+  font-weight: bold !important;
+}
```
Comment (javitocor) [inline Assets/style.css]: Please, do not use `!important`, it is only applicable in special cases. Kindly remove it

File: Assets/style.css [hunk 2]
```diff
@@ -0,0 +1,713 @@
...
+.funding-right-nav a:not(:last-of-type) {
+  border-right: 1px solid #f94640;
+}
+
+@media screen and (max-width: 767px) {
+  .funding-right-nav a {
+    margin: 5px 0;
+    border-right: 0 !important;
+  }
+}
```
Comment (javitocor) [inline Assets/style.css]: same here!

File: Assets/style.css [hunk 3]
```diff
@@ -0,0 +1,713 @@
...
+.funding-cell-footer {
+  font-size: 13px;
+  padding: 1rem 1rem 2rem 1rem;
+  background-color: #fffaf8;
+  border-top: 1px solid rgba(255, 68, 34, 0.25);
+}
+
+.funding-cell-footer .source {
+  color: #aaa !important;
+}
```
Comment (javitocor) [inline Assets/style.css]: Same here!

File: index.html [hunk 4]
```diff
@@ -0,0 +1,650 @@
...
+          <li class="mnl-list-item"><a class="mnl-link" href="#">HARD WORK</a></li>
+          <li class="mnl-list-item"><a class="mnl-link" href="#">PLUGGED</a></li>
+          <li class="mnl-list-item"><a class="mnl-link" href="#">README</a></li>
+          <li class="mnl-list-item"><a class="mnl-link" href="#">GROWTH HEADQUATERS</a></li>
+        </ul>
+        <ul class="main-nav-right">
+          <li class="mnr-list-item"><a class="mnr-link" href="#"><i class="fas fa-search"></i></a></li>
+          <li class="mnr-list-item"><a class="mnr-link" href="#"><i class="fas fa-bars"></i></a></li>
+          <li class="mnr-list-item"><a class="mnr-link" href="#"><img class="logo" src="images/logo.png" alt="logo" style="line-height:1"></a></li>
+          <li class="mnr-list-item"><a href="#" class="counter">00:20:30:20</a></li>
```
Comment (javitocor) [inline index.html]: Please do not use `inline styling`, you can move the style into the CSS file and improve your code's readability

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Hello team!
Amazing job on this project, I am impressed! 👏 
Good and descriptive commits and readme 👍 
Linters are green 🟢 
Good use of HTML tags 💯 
Clean and readable code 🥇 

However, we need to make some changes in order to get this project approved 💪 
Please, check my comments below 👇  apply the changes and resubmit the project for a new code review ⏩ 

## Changes
- In the main section, we should see the logo of the publication 'TNW', that disappears on medium screens.
    It contains TWN logo at the top left corner of the row ([link to page](https://thenextweb.com/))
    TWN logo disappears ([link to page](https://thenextweb.com/))
- Try to wrap all your media queries, I have counted 29 media queries. You should use only 2, one for 768px and another for 1024px
- The main section before the second breakpoint has a problem with the font size, kindly modify it
Capture
- In the Apps, Gear, Tech, etc... section, please add links to the list
Capture1
- And kindly modify the height of the main images in those sections to make it equal to the original
Original:
Capture3

Your project:
Capture2

- All your elements in your footer also must be links

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hello Team!! Great effort on the project!

## Required Changes
### [Mandatory] :red_circle: 

- [x] Section Latest funding is not matching the original in mobile version
image

- [x] Latest funding show an extra element in mobile version
image

- [x] The logo black rectagle does not match the navbar height
image

- [x] Elementes have extra white space
image

- [x] Latest deals are missing the correct information
image

- [x] Footer does not match original in mobile version
image

- [x] Social media does not match original style
image

- [x] Missing Margin between navbar and 3 image section
image

- [x] Latest funding does not match original
image

- [x] Navbar is nod displaying as original in tablet mode
image

- [x] Icons not displaying properly
image

### [Comments] :green_circle: 
- Is a great habit to use the Browser Media Query Inspector mode and the default values for media queries  1440, 1024, 768, 320 to check if the project looks as intended in different screen sizes. This also a great tool to test the original website and observer the changes.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Comment by zdrale** [comment]
> # Status: Changes Required 
> ##   Hello Team!! Great effort on the project!
> ## Required Changes
> ### [Mandatory] 
> * [ ]  Section Latest funding is not matching the original in mobile version
>   image
> * [ ]  Latest funding show an extra element in mobile version
>   image
> * [ ]  The logo black rectagle does not match the navbar height
>   image
> * [ ]  Elementes have extra white space
>   image
> * [ ]  Latest deals are missing the correct information
>   image
> * [ ]  Footer does not match original in mobile version
>   image
> * [ ]  Social media does not match original style
>   image
> * [ ]  Missing Margin between navbar and 3 image section
>   image
> * [ ]  Latest funding does not match original
>   image
> * [ ]  Navbar is nod displaying as original in tablet mode
>   image
> * [ ]  Icons not displaying properly
>   image
> 
> ### [Comments] 
> * Is a great habit to use the Browser Media Query Inspector mode and the default values for media queries  1440, 1024, 768, 320 to check if the project looks as intended in different screen sizes. This also a great tool to test the original website and observer the changes.
> 
> ## Keep up the great work  and Happy Coding  !!

>

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

Wow!! it's looking awesome! :clap: :clap: 

## Requested changes
## :wave: :wave: Hello!! Great effort!!

### [Mandatory] :red_circle: 

- [x] The live preview link is not Updated. This creates the appearance that the previously required changes have not been made.

### [Comments] :green_circle: 

- The project shows as intended in firefox. still in web browsers like chrome or opera, and probably safari is having a horizontal scrolling effect. This should not be happening. This might be something related to a CSS property that needs the vendor prefix for the specific property.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Comment by cyonii** [comment]
Thank you for your thorough and informative review.

We cannot seem to notice the horizontal scroll effect you commented on, we have tested in Firefox, Chrome, and Safari on both desktop and mobile. Could you be any clearer?

**Review by javitocor — APPROVED** [review]
# STATUS APPROVED 🥇 ⭐ 

Great job in this project, team! 👍 
You have met all the requirements 👏  and you can move into the next project! ⏩ 
Congratulations! 🥳 
Remember to fill up the completion form 📑  on your dashboard

Good luck! 😃

**Comment by jubaan** [comment]
> Thank you for your thorough and informative review.
> 
> We cannot seem to notice the horizontal scroll effect you commented on, we have tested in Firefox, Chrome, and Safari on both desktop and mobile. Could you be any clearer?

To reproduce it use the Media Query Inspector, (do not resize the browser window).

[Video of the issue](https://www.loom.com/share/af2a65c6c17942edb219d8ab288132bd)

**Pull request merged**